test/OpenTelemetry.Tests/Shared/CustomTextMapPropagator.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="CustomTextMapPropagator.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using OpenTelemetry.Context.Propagation;
+
+namespace OpenTelemetry.Tests;
+
+internal sealed class CustomTextMapPropagator : TextMapPropagator
+{
+#pragma warning disable SA1010
+    public List<string> ExtractValues = [];
+    public Dictionary<string, Func<PropagationContext, string>> InjectValues = [];
```

Comment:
[SA1010: OpeningSquareBracketsMustBeSpacedCorrectly](https://documentation.help/StyleCop/SA1010.html)

Is the pragma warning disable because of the List and Dictionary initializations (Lines 25 & 26)?
Just curious if the pragma disable could be removed if we used `= new()` instead?

Reply:
@TimothyMothra : 
Then I get the following compiler warning: 
IDE0028 (Simplify collection initialization)